wagtail/test/numberformat.py

```diff
@@ -17,6 +19,34 @@
 
 original_numberformat = django.utils.numberformat.format
 original_intcomma = django.contrib.humanize.templatetags.humanize.intcomma
+OriginalVariableNode = django.template.base.VariableNode
+
+
+@contextmanager
+def ignore_numberformat(template_names):
+    class VariableNode(OriginalVariableNode):
```

Comment:
I ended up adding a context manager instead so we can specify the templates to ignore in each test. Not sure if that's overengineering, though. Let me know if you'd rather us have a global list of templates to ignore.